tests/framework/CodeInterfaceTests/CobraTF/CobraTFinterfaceTest1/case5-15.inp

```diff
@@ -0,0 +1,679 @@
+***********************************************************************************************
+*MAIN CONTROL DATA                                                                             
```

Comment:
are we OK with putting this input file in the repo?
are we OK from an export control point of view?